js/coinsbit.js

```diff
@@ -0,0 +1,735 @@
+                //             [  "46801.25196599",  "0.4"   ],
+                //             ...
+                //         ]
+                //     }
+                const priceKey = 0;
+                const amountKey = 1;
+                const timestamp = this.milliseconds ();
+                orderbook = this.parseOrderBook (response, symbol, timestamp, 'bids', 'asks', priceKey, amountKey);
+            } else if (method === 'book') {
+                request['offset'] = this.safeString (params, 'offset', 0);
+                // ATM, they need to call separately (unfortunately)
+                request['side'] = this.safeString (params, 'side', 'buy');
+                const requestBids = this.spotPublicGetBook (this.extend (request, params));
+                request['side'] = this.safeString (params, 'side', 'sell');
+                const requestAsks = this.spotPublicGetBook (this.extend (request, params));
+                const responses = await Promise.all ([requestBids, requestAsks]);
+                // FOR EACH (BUY/SELL) REQUEST, THE RESPONSE IS SIMILAR (JUST 'side' PROPERTY IS DIFFERENT BETWEEN THEM)
+                //     {
+                //         success: true,
+                //         message: "",
+                //         result: {
+                //             offset: 0,
+                //             limit: 2,
+                //             total: 334,
+                //             orders: [
+                //                 {
+                //                     id: 8620142523,
+                //                     left: "1.56772",
+                //                     market: "BTC_USDT",
+                //                     amount: "1.56772",
+                //                     type: "limit",
+                //                     price: "46752.63032535",
+                //                     timestamp: 1640031161.544,
+                //                     side: "buy",
+                //                     dealFee: "0",
+                //                     takerFee: "0",
+                //                     makerFee: "0",
+                //                     dealStock: "0",
+                //                     dealMoney: "0"
+                //                 },
+                //                 ...
+                //             ]
+                //         },
+                //         code: 200
+                //     }
+                const resultBids = this.safeValue (responses[0], 'result', {});
+                const resultAsks = this.safeValue (responses[1], 'result', {});
+                const bids = this.safeValue (resultBids, 'orders', []);
+                const asks = this.safeValue (resultAsks, 'orders', []);
+                const response = { 'bids': bids, 'asks': asks };
+                const priceKey = 'price';
+                const amountKey = 'amount';
+                const timestamp = this.milliseconds ();
```

Comment:
Let's only take the timestamp from their response, we don't fill it to `this.milliseconds ()` by default.

Reply:
done

Reply:
Ok, though, note that there's no need to post `"done"` cause it will be automatically resolved when it's done.

Reply:
seen your comment now - sry, will not post 'done' more :)